pkg/pillar/zfs/zfs.go

```diff
@@ -195,6 +196,26 @@ func alignUpToBlockSize(size uint64) uint64 {
 	return (size + volBlockSize - 1) & ^(volBlockSize - 1)
 }
 
+//RemoveVDev removes vdev from the pool
+func RemoveVDev(log *base.LogObject, pool, vdev string) (string, error) {
```

Comment:
Are you not adding OfflineVdev and OnlineVdev functions ?

Reply:
They are available inside imported library (github.com/bicomsystems/go-libzfs), so no needed here.